public/js/index.js

```diff
@@ -1,5 +1,32 @@
-function displayMessage() {
-  return 'Hello, World!';
+function buildProduct(product) {
+  var div = document.createElement("div");
```

Comment:
This method also contains to many lines to read easily. You should split this methods content up in different methods, like : 
- createProductDiv
- setProductTitle
- setProductPicture
- ...